### PR TITLE
Add share options

### PR DESCRIPTION
**Pull request by adriaandotcom**
To give a bit of context, these are the parts that did change

On the home page:

<kbd>Screenshot</kbd>

After filling in the form and writing down the passcode:

<kbd>Screenshot</kbd>

Sharing on WhatsApp:

<kbd>Screenshot</kbd>

Sharing on Twitter:

<kbd>Screenshot</kbd>

Added all translations via Google Translate.

# Translations needed

## Text

- CALL TO ALL US CITIZENS: This is how we defeat the corona virus: together we make the invisible virus visible.
- Contribute and share {{ hostname }} on
- Fill out this form - it takes 1 minute - and help the United States gain control over COVID-19.
- One more thing
- To defeat the corona virus we need to make the invisible virus visible.
- {{ amount }} people already shared their health status.
- Let your friends fill out this form too and help the United States gain control over COVID-19.

## Languages done

If you don't know how to add these changes in this PR, just write the translations in the comments.

- [ ] .bd – Bangladesh (@redikod)
- [x] .ca – Canada (@mepa1363)
- [ ] .dk – Denmark (@alny)
- [x] .en-AU – Australia (@bitbonsai)
- [x] .en-IN – India
- [x] .en-MT – Malta (@tkrunning)
- [x] .en-NG – Nigeria (@desoga10)
- [x] .en-SG – Singapore (@jinjie)
- [x] .en-US – United States (@zacel)
- [x] .en – United Kingdom (@amritnagi)
- [x] .es-AR – Argentina (@waltergammarota)
- [x] .es-CL – Chili (@rubentd)
- [x] .es-CO – Colombia (@julianbetancourt)
- [x] .es-ES – Spain (@jesgarciamartinez)
- [x] .es-MX – Mexico (@alexalmadav)
- [x] .fr-FR – France (@bertrandlalo)
- [ ] .it – Italy (@zerbfra)
- [x] .ms-MY – Malaysia (@jinjie)
- [x] .nl – The Netherlands (@adriaanvanrossum @Kilian h@NickolasBoyer @julianbetancourt)
- [x] .no – Norway (@michaelmcmillan @fossecode)
- [x] .se – Sweden (@pontus @mihai-sysbio @MalinAhlberg @talavis)
- [x] .sk – Slovakia (sbocinec)
- [x] .tr – Turkey (@btutal)
- [x] .ua – Ukraine (@tarasmatsyk)
- [ ] .pt-PT – Portugal (@andrezzoid @Painatalman)
- [ ] .de-DE – Germany (@btutal @lseelenbinder)
- [ ] .ch-DE – Switzerland (@lseelenbinder)

### CODE REVIEW

File: app/routes/report-routes.ts [hunk 1]
```diff
@@ -34,6 +34,11 @@ const reportRepo = new CovidReportRepository();
 const passcodeCreator = getPasscodeCreator();
 
 router.get('/', async (req, res) => {
+  // return res.render('pages/confirm-profile', {
```
Comment (michaelmcmillan) [inline app/routes/report-routes.ts]: Delete these `// `? :)

**Review by michaelmcmillan — COMMENTED** [review]
This PR is on 🔥, really easy to review! I think @fossecode should take a quick look as well, but this is really good.

File: app/routes/report-routes.ts (hunk 1 above)
Comment (adriaandotcom) [reply app/routes/report-routes.ts]: oops

File: app/config.ts [hunk 2]
```diff
@@ -29,7 +29,8 @@ const fallbackConfig: Config = {
   REDIRECT_TO_GOVERNMENT:
     process.env.REDIRECT_TO_GOVERNMENT === 'true' || false,
   PASSCODE_LENGTH: parseInt(process.env.PASSCODE_LENGTH || '3', 10),
-  DB_PATH: process.env.DB_PATH || './covid_db'
+  DB_PATH: process.env.DB_PATH || './covid_db',
+  THOUSAND_SEPARATOR: process.env.THOUSAND_SEPARATOR || ' '
```
Comment (fossecode) [inline app/config.ts:33]: Maybe we could use the locale for this instead?

Comment (fossecode) [reply app/config.ts:33]: Or maybe config is easier :)

Comment (adriaandotcom) [reply app/config.ts:33]: Yeah, maybe easier. Default is more often `,` I think, but be let's keep it like this. Why not.

**Comment by fossecode** [comment]
Should we have a link to the profile page from the thank you page as well? :)

**Comment by adriaandotcom** [comment]
> Should we have a link to the profile page from the thank you page as well? :)

I omitted that on purpose because I rather have people stuck at sharing than automatically clicking to the homepage. Also, on the homepage is nothing new for them. So if they navigate they probably use the menu to go to stats or map.

**Comment by sbocinec** [comment]
Is there a better way how to push directly to this PR, than to create another PR to this PR? :-) https://github.com/BustByte/coronastatus/pull/463

**Comment by adriaandotcom** [comment]
@sbocinec you now have push access!

File: app/locales/es-ES.json [hunk 3]
```diff
@@ -181,11 +181,11 @@
...
+  "Contribute and share {{ hostname }} on": "Comparta {{ hostname }} en",
+  "Fill out this form - it takes 1 minute - and help the United States gain control over COVID-19.": "Complete este formulario (se tarda 1 minuto) y ayude a España a controlar el COVID-19.",
+  "One more thing": "Una cosa más",
+  "To defeat the corona virus we need to make the invisible virus visible.": "Para vencer al coronavirus necesitamos hacer visible el virus invisible.",
   "{{ amount }} people already shared their health status.": "{{ amount }} personas ya compartieron su estado de salud.",
-  "Let your friends fill out this form too and help the United States gain control over COVID-19.": "Deje que sus amigos llenen este formulario también y ayude a España a obtener el control sobre COVID-19."
+  "Let your friends fill out this form too and help the United States gain control over COVID-19.": "Dígale a sus amigos que llenen este formulario también y ayude a España a controlar el COVID-19."
```
Comment (jesgarciamartinez) [inline app/locales/es-ES.json]: In Spain we would say "rellenar el formulario", not "llenar", so the line would be:

"Let your friends fill out this form too and help the United States gain control over COVID-19.": "Dígale a sus amigos que rellenen este formulario también y ayude a España a controlar el COVID-19."

Not sure in other countries.

Comment (julianbetancourt) [reply app/locales/es-ES.json]: Hey man, it was me who made the change for all Spanish versions, sorry if the translation wasn't accurate enough for your country. Please feel free to open a PR to adjust it as it has not been merged yet

**Comment by tarasmatsyk** [comment]
UA:

```
CALL TO ALL US CITIZENS: This is how we defeat the corona virus: together we make the invisible virus visible.
Contribute and share {{ hostname }} on
Fill out this form - it takes 1 minute - and help the United States gain control over COVID-19.
One more thing
To defeat the corona virus we need to make the invisible virus visible.
{{ amount }} people already shared their health status.
Let your friends fill out this form too and help the United States gain control over COVID-19.

Прохання до усіх громадян: Так ми можемо перемогти коронавірус: разом ми робимо невидимий вірус - видими
Допомогти та поділитися {{ hostname }} у
Заповнення форми займає 1 хвилину і допомагає України розуміти стан поширення COVID-19
Іще одне
Щоб перемогти вірус ми повиині зробити невидимий вірус - видимим
{{ amount }} осіб заповнило анкети.
Попросіть друзів заповнити цю анкету і допомогти України розуміти стан поширення COVID-19 
```

File: app/locales/es-ES.json (hunk 3 above)
Comment (adriaandotcom) [reply app/locales/es-ES.json]: Fixed this for `app/locales/es-ES.json`

**Comment by btutal** [comment]
Turkish is translated.

**Comment by fossecode** [comment]
@pontus Could you check the swedish translation here? :)

**Comment by adriaandotcom** [comment]
Thank you @pontus!

**Comment by pontus** [comment]
But now it seems there was a few duplicates, apologies (I'm not really sure what happened there). But those should be non-breaking so we can clean up later, right?

**Comment by fossecode** [comment]
> But now it seems there was a few duplicates, apologies (I'm not really sure what happened there). But those should be non-breaking so we can clean up later, right?

Yep, dupes won't break anything

**Review by fossecode — APPROVED** [review]

**Comment by joanabarradas** [comment]
.pt-PT – Portugal

- Chamada de atenção a todos os cidadãos portugueses: É assim que vencemos o coronavírus: juntos tornamos visível, o vírus invisível.
- Contribui e partilha {{hostname}} no
- E mais uma coisa
- Para vencermos o coronavírus, precisamos tornar visível, o vírus invisível.
- {{ amount }} pessoas já partilharam o seu estado de saúde.
- Pede aos teus familiares e amigos para preencherem este formulário de forma a ajudar Portugal a ganhar controlo sob o COVID-19.

**Comment by adriaandotcom** [comment]
Thanks @joanabarradas, added those!

**Pull request merged**